src/main/converter/opds.ts

```diff
@@ -26,118 +25,211 @@ import { OPDSPublication } from "@r2-opds-js/opds/opds2/opds2-publication";
 
 // Logger
 const debug = debug_("readium-desktop:main/converter/opds");
+debug("opds-converter");
+
+interface IGetLinksViewFilter {
+    rel?: string | string[] | RegExp;
+    type?: string | string[] | RegExp;
+}
+
+const supportedFileTypeLinkArray = [
+    "application/epub+zip",
+    "application/vnd.readium.lcp.license-1.0+json",
```

Comment:
`application/vnd.readium.lcp.license-1.0+json`
should be:
`application/vnd.readium.lcp.license.1.0+json`
(`-` replaced with `.`)

Specification: https://readium.org/lcp-specs/releases/lcp/latest#32-content-conformance

Reply:
https://github.com/readium/readium-desktop/pull/832/commits/f692960474cbdc91b7677fe8950aecac7727f78c